<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTableDialogForm</class>
 <widget class="QDialog" name="CreateTableDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create table</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#CreateTableDialogForm
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(3, 252, 202, 255), stop:1 rgba(137, 54, 214, 255));
}</string>
  </property>
  <widget class="QWidget" name="FrameNearButtons" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>161</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#FrameNearButtons {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(2, 177, 139, 255), stop:1 rgba(110, 43, 172, 255));
	border-radius: 10px
}</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>49</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLineEdit" name="tableName">
       <property name="styleSheet">
        <string notr="true">border-radius: 5px;
background-color: rgb(31, 31, 31);
color: rgb(240, 240, 240);
border: 1px solid rgb(31, 31, 31);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="numberOfColumns">
       <property name="styleSheet">
        <string notr="true">border-radius: 5px;
background-color: rgb(31, 31, 31);
color: rgb(240, 240, 240);
border: 1px solid rgb(31, 31, 31);</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color: rgb(31, 31, 31);
color: rgb(240, 240, 240);
border: 1px solid rgb(31, 31, 31);</string>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QLabel" name="tableNameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Table name</string>
    </property>
   </widget>
   <widget class="QLabel" name="columnsLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Columns</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
